### DISCUSSION TITLE
Support module aliases in import statements by utilizing jiti's `alias` option

### DISCUSSION THREAD

**Discussion by philbaker4** [tailwindlabs/tailwindcss#11914] [Ideas]
Ideally, tailwind configs would utilize module aliases when importing custom plugins, theme overrides, etc. There's a [previously closed issue](https://github.com/tailwindlabs/tailwindcss/issues/11097) documenting jiti's isolated environment approach. However, it does seem that jiti supports an `alias` option through which you could pass import aliases. Given that it is likely not preferable for tailwind to support all of the different methods through which module aliases might be defined in a repository, my brain jumps to enabling a new `aliasPaths` property in tailwind configs that would be passed through to jiti's `alias` option. I'm certain I'm glossing over important details but wanted to restart this conversation to see if there's any progress to be made in this direction.

**Comment by philbaker4** [comment]
My current particular use case is importing a custom plugin and themes from [Nx](https://nx.dev/getting-started/intro) libraries using the defined alias for those libraries (in tsconfig `paths` option). While I can absolutely reference the library itself, it requires overriding eslint rules and breaks pattern and functionality within Nx's ecosystem.

**Comment by amirrezaDev1378** [comment]
any updates on this?
